octoprint_mrbeam/printing/comm_acc2.py

```diff
@@ -2079,29 +2079,50 @@ def _replace_intensity(self, cmd):
         obj = self._regex_intensity.search(cmd)
         if obj is not None:
             intensity_limit = int(self._laserCutterProfile["laser"]["intensity_limit"])
+            max_correction_factor = float(
+                self._laserCutterProfile["laser"]["max_correction_factor"]
+            )
             intensity_cmd = cmd[obj.start() : obj.end()]
             parsed_intensity = int(intensity_cmd[1:])
 
             # Limit GCode input (in case users enter a too high value in the gcode)
             self._current_intensity = parsed_intensity
             if self._current_intensity > intensity_limit:
+                self._logger.warn(
+                    "gcode intensity higher as allowed max, will limit to max value - %s => %s",
+                    self._current_intensity,
+                    intensity_limit,
+                )
                 self._current_intensity = intensity_limit
 
             # Apply power correction factor and limit again (in case there is something wrong with the calculation of
             # the correction factor)
+            if self._power_correction_factor > max_correction_factor:
+                self._logger.warn(
+                    "Power correction factor higher as allowed max, will limit to max value - %s => %s",
+                    self._power_correction_factor,
+                    max_correction_factor,
+                )
+                self._power_correction_factor = max_correction_factor
```

Comment:
Do we need a warning for _every single_ gcode command that goes over that max? I would recommend just to do

```python
self._power_correction_factor = min(self._power_correction_factor, max_correction_factor)
```

Reply:
If we don't log it we won't see why it is not the expected value, how about I change it to a debug level, so if we investigate it we could see what happens